code/modules/mining/ores_coins.dm

```diff
@@ -15,29 +15,29 @@
 	singular_name = "ore chunk"
 	var/points = 0 //How many points this ore gets you from the ore redemption machine
 	var/refined_type = null //What this ore defaults to being refined into
-	novariants = FALSE // Ore stacks handle their icon updates themselves to keep the illusion that there's more going
+	novariants = TRUE // Ore stacks handle their icon updates themselves to keep the illusion that there's more going

```

Comment:
#35772 does this too

Reply:
that's fine, this'll just get ignored on the merge since that one was merged already